content/docs/components-and-props.md

```diff
@@ -152,7 +152,7 @@ function Comment(props) {
 }
 ```
 
-[](codepen://components-and-props/extracting-components)
+**[Try it on CodePen](https://codepen.io/gaearon/pen/VKQwEo?editors=1010)**
```

Comment:
We are missing one link in this page

Reply:
https://codepen.io/gaearon/pen/rrJNJY is this the link we are missing ?

Reply:
Yes 💯